### ISSUE TITLE
Sum data not rendering.

### ISSUE THREAD

**Issue by Pupyrinth** [benjypng/logseq-tablerender-plugin#6]
I basically want to use the plugin as it's shown in the first gif, to see the sum of the numbers I add into some of the cells. As this is a way I'm using to track expenses.
But when I don't have the "nosum" it doesn't render anything.

(The second image is just to show that the renderer text is actually there).
![image](https://user-images.githubusercontent.com/47090855/157997871-4c1276f1-7d03-4ce5-9c80-0063e9954dae.png)
![image](https://user-images.githubusercontent.com/47090855/157997899-ab584897-5e07-40e2-b36a-1d3ebbc50a9a.png)

Here's some errors that I seem to get when I click away from the renderer text to let it render the table etc
```
Uncaught (in promise) ReferenceError: blockData is not defined
    at index.1e29aaaa.js:2
    at g (index.1e29aaaa.js:2)
    at r (index.1e29aaaa.js:2)
    at Ie (index.1e29aaaa.js:2)
    at e.t.render (index.1e29aaaa.js:2)
    at e.t.read (index.1e29aaaa.js:2)
    at Object.o (index.1e29aaaa.js:2)
    at k.<anonymous> (index.1e29aaaa.js:2)
(anonymous) @ index.1e29aaaa.js:2
g @ index.1e29aaaa.js:2
r @ index.1e29aaaa.js:2
Ie @ index.1e29aaaa.js:2
t.render @ index.1e29aaaa.js:2
t.read @ index.1e29aaaa.js:2
o @ index.1e29aaaa.js:2
(anonymous) @ index.1e29aaaa.js:2
async function (async)
(anonymous) @ index.1e29aaaa.js:2
s.emit @ index.1e29aaaa.js:2
#lspmsg# @ index.1e29aaaa.js:2
(anonymous) @ index.1e29aaaa.js:2
```

Please let me know if there's anything else you'd need me to give you 😃

**Comment by benjypng** [comment]
Apologies. This is fixed in the latest release (v1.8.1).

**Comment by Pupyrinth** [comment]
Yup, works perfectly now!
Thank you, awesome plugin!
